src/main/java/gov/usgs/wma/waterdata/FieldVisitDao.java

```diff
@@ -26,8 +29,14 @@
 	private Resource deleteDiscreteGroundWaterData;
 
 	@Transactional
-	public int doInsertDiscreteGroundWaterData(Long jsonDataId) {
-		return jdbcTemplate.update(getSql(insertDiscreteGroundWaterData), jsonDataId);
+	public List<FieldVisit> doInsertDiscreteGroundWaterData(Long jsonDataId) {
+		List<FieldVisit> fieldVisitList = Arrays.asList();
+		try {
+			fieldVisitList =  jdbcTemplate.query(getSql(insertDiscreteGroundWaterData), new FieldVisitRowMapper(), jsonDataId);
```

Comment:
Using .query might cause transactional issues - It's probably ok here since nothing happens after the insert, but the https://github.com/usgs/aqts-capture-ts-corrected/blob/master/src/main/java/gov/usgs/wma/waterdata/JsonDataDao.java got changed to use updates. That code did have the luxury of not needing to return values...